flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/result/BasicResult.java

```diff
@@ -50,4 +50,12 @@ public String getWebInterfaceUrl() {
 		}
 		return webInterfaceUrl;
 	}
+
+	/**
+	 * Result types.
```

Comment:
Make this a useful Javadoc and explain the difference between those two types.

Reply:
OK